hsts.permissions.yml

```diff
@@ -0,0 +1,3 @@
+administer strict transport security:
```

Comment:
This is actually left over from my newbiness in the 6.x and 7.x versions. Can you change the permission machine name to be "administer hsts". That way we don't collide with other permissions / way easier to type when checking permissions.